test/index.js

```diff
@@ -32,7 +32,9 @@ var getSiteInfo = function(domain) {
 };
 
 describe('Validation', function () {
-    var invalidTitles = [['', 'title-invalid-empty'],
+    var invalidTitles = [
+        ['foo�|bar', 'title-invalid-characters'],
```

Comment:
@subbuss Hm, weird artefact it github interface, the test is changed, look at the actual file